Makefile

```diff
@@ -49,3 +50,33 @@ bin/confd: $(GO_FILES) vendor/.up-to-date
 	    sh -c 'go build -v -i -o $@ "github.com/kelseyhightower/confd" && \
                ( ldd bin/confd 2>&1 | grep -q "Not a valid dynamic program" || \
 	             ( echo "Error: bin/confd was not statically linked"; false ) )'
+
+.PHONY: test-kdd
+## Run template tests against KDD
+test-kdd: bin/confd run-etcd-host run-k8s-apiserver
```

Comment:
Looking for feedback on that, right now I just built it manually and reference what I tagged it as here for testing.  I'll add a make target to build it, but I'm not sure if we want to add that as a dependency to the test targets, or have the test targets just pull down a version of the container.  If we go that route we would need to mount in `bin/confd` to the container so we can access the latest binary.